### DISCUSSION TITLE
How to get familiarized with the code structure

### DISCUSSION THREAD

**Discussion by waterflow80** [joomla/joomla-cms#39849] [Q&A]
Hello,

I'm a software engineering student and I am really interested in contributing to **Joomla-cms**.
Are there any small issues that I can start with in order to get familiarized with the code structure,
so I can make more complex contributions in the future.

Thank you.

**Comment by chmst** [comment]
The way is learning by doing. Join us on mattermost https://joomlacommunity.cloud.mattermost.com/main/channels/gsoc2023 :)
